specification/storageimportexport/resource-manager/Microsoft.ImportExport/stable/2016-11-01/storageimportexport.json

```diff
@@ -143,7 +143,8 @@
             "in": "query",
             "description": "An integer value that specifies how many jobs at most should be returned. The value cannot exceed 100.",
             "required": false,
-            "type": "integer"
+            "type": "integer",
+            "format": "int64"
```

Comment:
as by default in SDK is int32.  if int32 is enough for your service, can we set format int32?

Reply:
yes I think that should be good, will make the change